nucleus/model.py

```diff
@@ -109,9 +116,10 @@ def __init__(
         self.bundle_name = bundle_name
         self.tags = tags if tags else []
         self._client = client
+        self.trained_slice_id = trained_slice_id
 
     def __repr__(self):
-        return f"Model(model_id='{self.id}', name='{self.name}', reference_id='{self.reference_id}', metadata={self.metadata}, bundle_name={self.bundle_name}, tags={self.tags}, client={self._client})"
+        return f"Model(model_id='{self.id}', name='{self.name}', reference_id='{self.reference_id}', metadata={self.metadata}, bundle_name={self.bundle_name}, tags={self.tags}, client={self._client}, trained_slice_id={self.trained_slice_id})"
```

Comment:
I'd drop this from the repr. Not commonly used and it is already way too long.

Reply:
The only issue with removing it is that it fails the repr comparison test.